ProcessLib/HT/HTProcess.cpp

```diff
@@ -160,5 +165,54 @@ void HTProcess::setCoupledTermForTheStaggeredSchemeToLocalAssemblers()
         _local_assemblers, _coupled_solutions);
 }
 
+NumLib::LocalToGlobalIndexMap* HTProcess::getDOFTableForExtrapolatorData(
+    bool& manage_storage) const
+{
+    if (!_use_monolithic_scheme)
+    {
+        // For single-variable-single-component processes reuse the existing DOF
+        // table.
+        manage_storage = false;
+        return _local_to_global_index_map.get();
+    }
+
+    // Otherwise construct a new DOF table.
+    std::vector<MeshLib::MeshSubsets> all_mesh_subsets_single_component;
+    all_mesh_subsets_single_component.emplace_back(
+        _mesh_subset_all_nodes.get());
+
+    manage_storage = true;
+
+    return new NumLib::LocalToGlobalIndexMap(
```

Comment:
Please use automatic storage management for on-heap allocated objects.

Reply:
Explained in the discussion.